arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -124,6 +124,24 @@ Or:
   chmod -fR 777 /data/arbitrum
   ```
 
+### Note on Sequencer feed
+
+The Arbitrum node requires a feed to receive real time ordered transactions from the sequencer. If you don't set feed input url, your node will listen to parent chain inbox contract to get ordered transaction which will cause your node to be unable to synchronize the latest state.
+
+To make your node listen to sequencer feed, you need to set those configurations to your sequencer node:
+  
+  ```shell
+  --node.feed.output.enable=true --node.feed.output.addr=<Sequencer feed url> --node.feed.output.port=<Sequencer feed port>
+  ```
+
+Then set the following configurations to your new node:
```

Comment:
I would maybe put these before the ones for the sequencer. I'm not sure though, but I'm thinking that this document will be mainly read by users that want to run a full node (and not chain owners who can modify the config of the sequencer) 🤔